rocketpool-cli/pdao/commands.go

```diff
@@ -1059,7 +1059,7 @@ func RegisterCommands(app *cli.App, name string, aliases []string) {
 									{
 										Name:      "submit-balances-frequency",
 										Aliases:   []string{"sbf"},
-										Usage:     fmt.Sprintf("Propose updating the %s setting; %s", protocol.SubmitBalancesFrequencySettingPath, blockCountUsage),
+										Usage:     fmt.Sprintf("Propose updating the %s setting; %s", protocol.SubmitBalancesFrequencySettingPath, uintUsage),
```

Comment:
We probably want this (and `SubmitPricesFrequencySettingPath` below) to be durations instead of uints; since the actual value is going to be in seconds, duration makes more sense here.